script/open-in-chrome

```diff
@@ -2,18 +2,20 @@
 
 // usage: open-in-chrome [extension-dir] [starting-url]
 
+/* eslint-disable import/no-extraneous-dependencies */
+
 const path = require('path');
 const launcher = require('chrome-launcher');
 
 const dir = process.argv[2] || path.join(__dirname, '..', 'dist', 'chrome');
 const url = process.argv[3] || 'https://github.com/bitcrowd/tickety-tick';
 
-const chromeFlags = launcher.default.defaultFlags()
-  .filter(flag => flag !== '--disable-extensions')
+const chromeFlags = launcher.Launcher.defaultFlags()
```

Comment:
New API as per the [updated documentation](https://github.com/GoogleChrome/chrome-launcher/tree/2757e0618e89506bb0aa5c2abc603bc5005f667f#launching-with-support-for-extensions-and-audio).